source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Vulnerability Disclosure and security.txt
+last_reviewed_on: 2021-04-09
+review_in: 6 months
+---
+# <%= current_page.data.title %>
+
+## Vulnerability Disclosure
+
+In the Cabinet Office, including GDS, the [CDIO cyber security team] run a
+vulnerability disclosure programme with HackerOne and a third party to triage
+reports from security researchers.
+
+This isn't a sign-post for security reseachers to 'hack' our systems but we want
+to advocate secure disclosure so we can find out about issues and fix them
+before they cause a security incident.
+
+GOV.UK hosts the security policy:
+<https://www.gov.uk/help/report-vulnerability>
+
+## security.txt
+A `security.txt` file is a way of telling researchers how to get in contact with
+us. As per the current policy, we only accept reports from services that have a
+`security.txt` file pointing the [security policy].
+
+The file should be hosted at `https://{domain}/.well-known/security.txt`, use a
```

Comment:
We should try to avoid the passive voice here. Who makes the decision to host the file?

Reply:
@cmcnallygds - do the latest changes work?

Reply:
yep - have added a few more comments!